flask_cors/__init__.py

```diff
@@ -544,3 +561,12 @@ def _serialize_options(options):
 
     if isinstance(options.get('max_age'), timedelta):
         options['max_age'] = str(int(options['max_age'].total_seconds()))
+
+
+def getLogger(app=None):
```

Comment:
Catching exceptions in Python is still relatively expensive, Flask exposes the context stack, and @mitsuhiko's Flask-SQLAlchemy uses it: https://github.com/mitsuhiko/flask-sqlalchemy/blob/3d3c72faeef2f0195397cf84e788924395d280e9/flask_sqlalchemy/__init__.py#L250
I see no reason to break the function apart.